README.md

```diff
@@ -182,6 +182,31 @@ related:
 ```
 
 
+## nugetAlias.txt
+
+All NServiceBus related nugets (used in documentation) are listed in [components/nugetAlias.txt](components/nugetAlias.txt). The alias part of the nuget is the key that is used to infer the version and component for all snippets. So for example [Snippets/Callbacks](Snippets/Callbacks) has, over its lifetime, existed in both the Core nuget and the Callbacks nuget. So the directories under Callbacks are indicative of the nuget (alias) they exist in and then split over the multiple versions of a given nuget.
+
+
+## components.yaml
+
+"Components" is a general terms used to describe a deployable set of functionality. Components exist in [components/components.yaml](components/components.yaml). Note that over time a Component may have moved between nugets or split into new nugets. For example the ABS DataBus or the Callbacks.
+
+
+### Component Key
+
+The Component Key allows for shorthand when referring to components in page headers.
```

Comment:
Little snippet/sample here of what you refer to would be nice for visualization.